week-2/extended-project/banzai/todo-list/cli.rb

```diff
@@ -0,0 +1,35 @@
+# define global variable
+$bd_name = File.expand_path('../', __FILE__) + '/bd.txt' 
+def read_file
+  File.read($bd_name).split("\n").map { |line| line.split(" - ").map(&:strip) }
+end
+
+def save_file(tasks)
+  newTasks = tasks.map { |fil|  fil.join(' - ') }.join("\n")
+  File.write($bd_name, newTasks)
+end
+
+def write(task)
+  list_task = read_file
```

Comment:
The name of the method `read_file` is confused.

Reply:
Would be better if we change it to *read_list_of_tasks* ?